lib/api/view.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * eq8js
+ * Copyright(c) 2016 Benjamin Bartolome
+ * Apache 2.0  Licensed
+ */
+
+'use strict';
+
+module.exports = function exported(_, async) {
+	return function view(query, urlPath, done) {
```

Comment:
TODO: change `urlPath` to `context` similar to https://github.com/eq8/eq8js/blob/develop/lib/api/act.js#L10